### PR TITLE
draft with routes

### PR DESCRIPTION
**Pull request by MaksOnofriychuk**

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -0,0 +1,129 @@
+html,
```
[the remark this answers is not in the document]
Comment (VladPobidash) [reply src/index.scss:1]: reset.scss as scss module

[the remark this answers is not in the document]
Comment (VladPobidash) [reply src/index.scss:1]: @import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');

File: src/components/Home/FirstScreen.jsx [hunk 2]
```diff
@@ -1,7 +1,19 @@
 import React from "react";
```
Comment (VladPobidash) [inline src/components/Home/FirstScreen.jsx:2]: Your next task. Here should be a full screen picture and h1 title with a "Buy" button

File: src/App.js [hunk 3]
```diff
@@ -1,5 +1,21 @@
...
+
+
 function App() {
-  return <div>Hello World</div>;
+  return (
+     <BrowserRouter>
+     <Navbar />
+     <Switch>
+         <Route path="/" exact component={Home} />
+        <Route path="/kits"  render={(props) => <Category {...props} categoryName="комплекти"/>} />
```
[the remark this answers is not in the document]
Comment (MaksOnofriychuk) [reply src/App.js:15]: I did through useRouteMatch

**Pull request merged**